commons/passive/dreem/dreem_sleep_endpoints.avsc

```diff
@@ -0,0 +1,69 @@
+{
+  "type": "record",
+  "namespace": "org.radarcns.passive.dreem",
+  "name": "DreemSleepEndpoints",
+  "doc": "AVRO Schema for DREEM sleep endpoints. These are statistics (metrics) calculated from DREEM Hypnogram",
+  "fields": [
+    {
+      "name": "time",
+      "type": "double",
+      "doc": "This refers to the recDateTime field from the dataset."
+    },
+    {
+      "name": "timeReceived",
+      "type": "double",
+      "doc": "This is the processing time in UTC (s)."
+    },
+    {
+      "name": "offHead",
+      "type": "float",
+      "doc": "Percentage of the total recording time TRT  during which the headband was on head"
+    },
+    {
+      "name": "endpoint",
+      "type": "enum",
+      "symbols": [
+        "TRT",
+        "TST",
+        "SE",
+        "SQL",
+        "WASO",
+        "RL",
+        "NREM",
+        "N1",
+        "N2",
+        "N3",
+        "REM",
+        "p_N1",
+        "p_N2",
+        "p_N3",
+        "p_REM",
+        "p_NREM",
+        "RR",
+        "RRN1",
+        "RRN2",
+        "RRN3",
+        "QUAL"
+      ],
```

Comment:
Instead of using an enum and providing just one value, I would prefer if you have one field per metric, and write out the metric name. If there is a qiIndex for each endpoint, you can also use a new record type DreemSingleMetric with only value and qiIndex fields and make the fields that type.